tests/lib/reset.sh

```diff
@@ -3,17 +3,11 @@
 set -e -x
 
 systemctl stop snapd.service snapd.socket
-mounts="$(systemctl list-unit-files | grep '^snap[-.].*\.mount' | cut -f1 -d ' ')"
-services="$(systemctl list-unit-files | grep '^snap[-.].*\.service' | cut -f1 -d ' ')"
-for unit in $services $mounts; do
-    systemctl stop $unit || true
-done
+
+# purge all state
+sh $PROJECT_PATH/debian/snapd.postrm purge
```

Comment:
Thanks for doing this, I should be able to reuse this in packaging for various other distributions